tensorrt_llm/_torch/modules/fused_moe/fused_moe_wide_ep.py

```
@@ -776,8 +786,8 @@ def split_chunk(split_token_num: int, split_num_chunks: int):
 
             outputs_list = []
             # Postpone reduce-scatter/all-reduce to the next iteration to achieve better overlap
-            for idx_chunk, (x, router_logits) in enumerate(
-                    zip(x_list, router_logits_list)):
+            for idx_chunk, (x, router_logits, input_ids_chunk) in enumerate(
+                    zip(x_list, router_logits_list, input_ids_list)):
```

Ruff (0.15.17)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)